src/main/webapp/scripts/civic.js

```diff
@@ -2,19 +2,31 @@
  * Extract the government officials from the civic API response.
  */
 function extractOfficials(civicObj) {
-  let officials = new Object();
-  const levels = ["administrativeArea1", "administrativeArea2", "country", "international", "locality", "regional", "special", "subLocality1", "subLocality2"];
-  levels.forEach((level) => {
-    officials[level] = new Array();
-  });
+  let levelsArray = new Array();
+  const levels = {"international" : "International",
+                  "country" : "Country",
+                  "regional" : "Regional",
+                  "administrativeArea2" : "Administrative Area 2",
+                  "administrativeArea1" : "Administrative Area 1",
+                  "locality" : "Locality",
+                  "subLocality2" : "Sublocality 2",
+                  "subLocality1" : "Sublocality 1",
+                  "special" : "Special",};
+  for (const level in levels) {
+    let levelsObj = new Object();
+    levelsObj.levelType = level;
+    levelsObj.levelName = levels[level]
+    levelsObj.officials = new Array();
+    levelsArray.push(levelsObj);
+  }
   civicObj.offices.forEach((officeObj) => {
     const level = officeObj.levels[0];
     // Loop through the officials Indices that work at the current office, and add them to their coresponding level.
```

Comment:
[Nit] I think this is clear from the code since you are using a forEach loop and the variable is name officialIdex.  In generally try to avoid comments that say _what_ the code does.  Comments should instead be used to explain _why_ the code is written in the way it is.